src/controllers/pokestop.js

```diff
@@ -13,8 +14,8 @@ class Pokestop extends Controller {
 		let query = `
 		select humans.id, humans.name, humans.type, humans.latitude, humans.longitude, invasion.template, invasion.distance, invasion.clean, invasion.ping from invasion
 		join humans on humans.id = invasion.id
-		where humans.enabled = true and
-		(invasion.grunt_type='${data.gruntType}' or invasion.grunt_type = 'everything') and
+		where humans.enabled = 1 and
+		(invasion.grunt_type='${data.gruntType.toLowerCase()}' or invasion.grunt_type = 'everything') and
```

Comment:
hmm, did you find gruntType not matching due to capitalization?

Reply:
Yes, exactly that (sqlite)